pkg/api/k8s_versions.go

```diff
@@ -85,7 +85,7 @@ var kubernetesImageBaseDefaultImages = map[string]map[string]string{
 		common.DashboardMetricsScraperContainerName: "oss/kubernetes/metrics-scraper:v1.0.4",
 		common.ExecHealthZComponentName:             "oss/kubernetes/exechealthz:1.2",
 		common.HeapsterAddonName:                    "oss/kubernetes/heapster:v1.5.4",
-		common.CoreDNSAddonName:                     "oss/kubernetes/coredns:1.6.7",
+		common.CoreDNSAddonName:                     "oss/kubernetes/coredns:1.6.9",
```

Comment:
It looks like updates to coredns are no longer being publicly hosted here:

https://console.cloud.google.com/gcr/images/google-containers/GLOBAL/coredns?gcrImageListsize=30

So, just updating the MCR source